lua/cmp/config/mapping.lua

```diff
@@ -178,6 +178,15 @@ mapping.scroll_docs = function(delta)
   end
 end
 
+-- Toggle the documentation window.
+mapping.toggle_docs = function()
+  return function(fallback)
+    if not require('cmp').toggle_docs() then
+      fallback()
+    end
+  end
+end
```

Comment:
I think it should be `open/close` instead of `toggle`.

Reply:
You mean I shouldn't create a new toggle function?
Note that I changed the implementation a bit in #1647, since I realized that this first attempt wouldn't properly update the documentation window when navigating through the list.